contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -46,6 +46,9 @@ install() {
 		# On systems with gcc-config (Gentoo, Funtoo, etc.):
 		# Use the current profile to resolve the appropriate path
 		dracut_install "/usr/lib/gcc/$(s=$(gcc-config -c); echo ${s%-*}/${s##*-})/libgcc_s.so.1"
+	elif [[ -n "$(ls /usr/lib/libgcc_s.so*)" ]]; then
```

Comment:
Let's redirect stderr to `/dev/null` to get rid of the error message when it doesn't exist.

```
elif [[ -n "$(ls /usr/lib/libgcc_s.so* 2>/dev/null)" ]]; then
```